web/js/scripts.js

```diff
@@ -102,9 +106,17 @@ var grouphub = (function ($) {
         });
     };
 
+    var updateGroupsInGroup = function () {
+        var $groups = $('.edit_group #group_in_group_tab ul.member_collection');
```

Comment:
This seems a bit too tightly bound to the DOM for my liking. There is no guarantee that this only returns 1 element, so the `$groups.data('url')` could possibly fail in the future. Not sure if this is something you should fix now, as this is how the rest of the front-end is built, but it would be nice to have the datamodel decoupled from the UI / DOM.